azurelinuxagent/common/osutil/default.py

```diff
@@ -105,6 +105,10 @@ def __init__(self):
         self.disable_route_warning = False
         self.jit_enabled = False
 
```

Comment:
It'll be better to defile service_name here, rather than defining in all the child classes.
```suggestion
        self.service_name = self.get_service_name()
```